packages/tocco-ui-showcase/src/toccoUtilExamples/notifier/example.js

```diff
@@ -19,11 +19,12 @@ class Example extends React.Component {
   }
 
   info = () => {
-    this.store.dispatch(notifier.info('info', 'client.title', 'client.description', 'info', 2000))
+    this.store.dispatch(notifier.info(
+      'info', <Typography.H4>Typography component</Typography.H4>, 'string contains <b>html</b>', 'info', 2000))
   }
 
   success = () => {
-    this.store.dispatch(notifier.info('success', 'client.title', 'client.description', 'thumbs-up', 0))
+    this.store.dispatch(notifier.info('success', 'client.title', 'Lorem Ipsum', 'thumbs-up', 2000))
```

Comment:
Why did you change the timeout to 2000? With 0 the toastr stays which could be used in some cases. If you think this should be the same every time we should rather remove that property.